src/components/footer/index.js

```diff
@@ -68,6 +69,11 @@ function Footer() {
                 <p>
                     {t("If you'd also like to support this project, you can make a donation and/or become a backer on")}{' '}
                     <a href="https://opencollective.com/tarkov-dev" target="_blank" rel="noreferrer">Open Collective</a>
+
+                    <br></br>
+                    <br></br>
+
+                    <OpenCollectiveButton />
```

Comment:
Instead of the two line breaks, why not just insert the OpenCollectiveButton element after the close of the paragraph tag like with the Ukraine button?

Reply:
Ah great suggestion! UI skillz are still 🥔 for me.